docs/src/features.md

```diff
@@ -172,3 +172,51 @@ julia> summarize(s, ns=2)
  "Assume this Short Document as an example."
  "This has too foo sentences."
 ```
+
+##Parts of Speech Tagger
```

Comment:
Can you try building the documentation locally? I feel that this should have a space in between `#` and heading, for mardown to take affect.

Reply:
Thanks @Ayushk4 for this